Assets/Scripts/PlayerFall.cs

```diff
@@ -10,60 +10,57 @@ public class PlayerFall : MonoBehaviour
     //speed at which the player is dragged down
     public float thrust = 10.0f;
     private GameObject Camera;
+    private GameObject TrapDoor;
     private GameObject Player;
     //height from Player we want Camera to be
     private float desiredCameraHeight = 5.0f;
     private Vector3 direction;
     private float timer = 0.0f;
     private float delayuntilFastFall = .5f;
     
-    //falling audio at this time overriden in BeginFall method
     [SerializeField]
     private AudioSource fallingAudio;
     
     //flag
-    public bool trapOpened {get; set;}
+    private bool trapOpened = false;
     
     void Start(){
         
-        GameObject.Find("TrapDoor").transform.GetChild(0).transform.GetChild(0).GetComponent<TrapDoorOpen>().TrapOpened += BeginFall;
-        GameObject.Find("TrapDoor").transform.GetChild(1).transform.GetChild(0).GetComponent<TrapDoorOpen>().TrapOpened += BeginFall;
+        Camera = GameObject.FindWithTag("MainCamera");
+        TrapDoor = gameObject;
+        TrapDoorOpen[] hinges = GetComponentsInChildren<TrapDoorOpen>();
+        foreach(TrapDoorOpen script in hinges)
+        script.TrapOpened += BeginFall;

```

Comment:
Possible bug (No need to fix now), if the player manages to trigger both hinges, then `BeginFall` method will run twice possibly causing some weird results.

Reply:
That happens because he subscribed to it twice on both hinges.

Reply:
true. will fix later